class ApplicationError(Exception):
    pass


class NotFoundError(ApplicationError):
    pass
